packages/block-library/src/table/edit.js

```diff
@@ -499,27 +473,14 @@ function TableEdit( {
 							onChange={ onToggleFooterSection }
 						/>
 					</PanelBody>
-					<PanelColorSettings
-						title={ __( 'Color' ) }
-						initialOpen={ false }
-						colorSettings={ [
-							{
-								value: backgroundColor.color,
-								onChange: setBackgroundColor,
-								label: __( 'Background color' ),
-								disableCustomColors: true,
```

Comment:
Previously custom colors were intentionally disabled for the table block. This PR enables custom colors for the table block. Is this change intentional? Should we keep them disabled using lib/experimental-default-theme.json?

Reply:
With the adoption of the colors block support increasing the user's control over colors including font color, it seemed like this was the preferred direction to go. More so when taking into account some of the conversation in https://github.com/WordPress/gutenberg/issues/19659.